nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-processors/src/main/resources/docs/org.apache.nifi.processors.websocket.ConnectWebSocket/additionalDetails.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8"/>
+    <title>ConnectWebSocket</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
+</head>
+
+<body>
+<h2>Summary</h2>
+<p>
+    This processor acts as a WebSocket client endpoint to interact with a remote WebSocket server.
+    It is capable of receiving messages from a websocket server and it transfers them to downstream relationships
+    according to the received message types. Dynamic properties and custom header attributes can be configured by
+    incoming
+    flowfile attributes.
+</p>
+<p>
+    Any flowfile attributes can be referenced in the processor and in its WebSocket Client Service. For dynamic
+    properties
+    use the NiFi Expression Language and reference a flowfile attribute. For instance to the "dynamic.url =
+    currentValue"
+    flowfile attribute can be referenced in the WebSocket Client Service with the ${dynamic.url} expression.
```

Comment:
```suggestion
    The processor may have an incoming relationship, in which case flowfile attributes are passed down to its WebSocket Client Service. This can be used to fine-tune connection configuration (url and headers for example). For example the "dynamic.url =
    currentValue"
    flowfile attribute can be referenced in the WebSocket Client Service with the ${dynamic.url} expression.
```